index.html

```diff
@@ -605,7 +605,224 @@ <h2>Assertion Method</h2>
+      of a DID document, there is always a way to represent a `service` or a `verificationMethod`, and their meanings
+      are consistent across conformant representations.
+    </p>
+
+    <p>
+      In addition to “properties”, a DID document may also contain “representation-specific entries” which may be
+      required by certain representations in order to be able to fully represent the DID document data model and its
+      properties. For example, the JSON-LD representation (with media type `application/did+ld+json`) requires the
+      `@context` representation-specific entry, in order to correctly express the semantics of properties such as `id`,
+      `verificationMethod` or `service` within the JSON-LD representation. This enables JSON-LD’s semantics which are
+      based on a decentralized, permissionless, open-world data model.
+    </p>
+
+    <p>
+      In contrast to this, the JSON representation (with media type `application/did+json`) does not have this semantic
+      capability; instead, (non-LD) JSON documents always need agreement on semantics in some out-of-band way that is
+      not part of the document itself. In the case of DID documents using the JSON representation, those semantics are
+      established by the <a href="https://w3c.github.io/did-spec-registries/">DID Spec Registries</a>. Therefore,
+      establishing semantics of DID document properties in the JSON representation does not require any
+      “representation-specific entries”, as is the case with the JSON-LD representation. The same is true for the CBOR
+      representation.
+    </p>
+
+    <p>
+      Other examples of “representation-specific entries” could be XML namespace definitions, or YAML tags, although
+      these particular representations have not been defined at the time of this writing.
+    </p>
+
+    <p>
+      If an entry that is specific to one representation appears in another representation, it can be called a
+      "representation-foreign entry". An example of this is an `@context` entry appearing in the `application/did+json`
+      representation.
+    </p>
+
+    <p>
+      Examples of “representation-specific entries” and “representation-foreign entries”:
+    </p>
+
+    <figure id="representation-entries">
+      <img style="margin: auto; display: block;"
+           src="diagrams/representation-specific-and-foreign-entries.png" alt="
+Examples of representation-specific and representation-foreign entries in various representations.">
+      <figcaption>
+        Representation-specific and representation-foreign entries.
+      </figcaption>
+    </figure>
+
+    <section class="informative">
+      <h3>
+        Avoid representation-foreign entries
+      </h3>
+
+      <p class="advisement">
+        It is bad practice and potentially harmful for producers to include "representation-foreign entries" in DID
+        documents.
```

Comment:
-1 to giving advice when there is not consensus on what advice to give. This should be reworded to say that this is one consideration. An "on the other hand" should be provided that suggests the other advice -- from Orie's text -- that (I presume) is essentially in opposition to this. If the WG could not get consensus on which is the better advice, we should not be giving advice either way but merely presenting the arguments and potential/predicted outcomes.